doctordroid/manifest.json

```diff
@@ -0,0 +1,52 @@
+{
+    "manifest_version": "2.0.0",
+    "app_uuid": "5e75658c-065e-460f-b9f8-42bf100e361d",
+    "app_id": "doctordroid",
+    "display_on_public_website": true,
+    "tile": {
+      "overview": "README.md#Overview",
+      "configuration": "README.md#Setup",
+      "uninstallation": "README.md#Uninstallation",
+      "support": "README.md#Support",
+      "changelog": "CHANGELOG.md",
+      "description": "Analyse your alerts to identify trends to improve noise and coverage",
+      "title": "Doctor Droid",
+      "media": [
+        {
+          "media_type": "image",
+          "caption": "Datadog Alert Occurence Trend",
+          "image_url": "images/alert_occurence_count_trends.png"
+        },
+        {
+          "media_type": "image",
+          "caption": "Datadog Top 5 alerts occurence distribution",
+          "image_url": "images/top_5_alerts_distribution.png"
```

Comment:
There seems to be overlapping in this image making it hard to read

Reply:
Just bumping this comment as I believe it's unresolved

Reply:
Updated the image